nextflow_schema.json

```diff
@@ -169,6 +169,20 @@
                 }
             }
         },
+        "sv_caller_options": {
+            "title": "Structural variant calling options",
+            "type": "object",
+            "fa_icon": "fas fa-map-signs",
+            "description": "Options to adjust parameters and filtering criteria for structural variant callers.",
+            "properties": {
+                "cnvpytor_chr": {
+                    "type": "string",
+                    "default": null,
+                    "description": "Specifies the chromosome for cnvpytor run",
```

Comment:
You could add a description on how they are supplied (comma separated list I guess i.e. 1,2,3...)

Reply:
You're right. The description in the test config file was this "specifies chromosome name(s) the same way as they are described in the sam/bam/cram header e.g. '1 2' or 'chr1 chr2 "  which I think looks more informative.